CMakeLists.txt

```diff
@@ -152,6 +153,27 @@ if (NOT TPP_INSIDE_IREE)
                     DEPENDS tpp-opt tpp-run xsmm_dnn_mlp
                     WORKING_DIRECTORY ${BENCHMARK_DIR}
                     COMMENT Run Benchmarks)
+
+  if (TPP_GPU)
+    if (TPP_GPU MATCHES "cuda")
+      set(BENCH_CUDA_CFGS
+        ${CONFIG_DIR}/GPU/cuda.json
+      )
+      set(BENCH_CUDA_DEPS
+        bench_cuda_matmul
+      )
+    endif()
+
+    string(JOIN ',' BENCH_GPU_CFGS_STR ${BENCH_CUDA_CFGS})
+    string(JOIN ' ' BENCH_GPU_DEPS_STR ${BENCH_CUDA_DEPS})
+
+    # Run a small set of benchmarks with small iterations to test the benchmarks and run locally on small machines
+    add_custom_target(quick-gpu-bench ${BENCHMARK_DIR}/driver.py -v --build ${PROJECT_BINARY_DIR} -n 10
```

Comment:
Guard this target with `BENCH_GPU_CFGS_STR` not being empty. Right now, non-cuda targets will make it empty

Reply:
Correct, it will just throw an error that it misses cfg file. Later we'll also have vulkan so there'll be always sth active.